FirstProject/assets/evaluation.less

```diff
@@ -34,11 +87,165 @@
 }
 
 a.border_bottom:hover {
-  border-bottom: 4px solid blue;
+  border-bottom: 4px solid @blue;
   border-bottom-width: 7px;
 }
 
 table {
   width: 100%;
 }
 
+/*Evaluation html*/
+
+.add_button {
```

Comment:
For the sake of consistency, I'd go with a `spinal-case` (aka `kebab-case`) naming convention, as opposed to `snake_case`. I do realize that some of the other classes below are 3rd party, but it's still nice to be able to have consistency throughout the codebase.